src/storage/txn/scheduler.rs

```diff
@@ -52,6 +52,17 @@ use crate::storage::{
     ErrorInner as StorageErrorInner,
 };
 
+use crate::storage::metrics::SCHED_POST_HANDLE_DURATIONS_VEC;
```

Comment:
I think you can use `use crate::storage::metrics::*;`. We allow glob import for metrics.

Reply:
Addressed